mysql-test/suite/sys_vars/t/binlog_space_limit_basic.test

```diff
@@ -0,0 +1,88 @@
+--source include/have_log_bin.inc
+#
+# only global
+#
+select @@global.binlog_space_limit;
+--error ER_INCORRECT_GLOBAL_LOCAL_VAR
+select @@session.binlog_space_limit;
+show global variables like 'binlog_space_limit';
+show session variables like 'binlog_space_limit';
+--disable_warnings
+select * from information_schema.global_variables where variable_name='binlog_space_limit';
+select * from information_schema.session_variables where variable_name='binlog_space_limit';
+--enable_warnings
+
+#
+# show that it's read-only
+#
+--error ER_INCORRECT_GLOBAL_LOCAL_VAR
+set global binlog_space_limit=1;
+--error ER_INCORRECT_GLOBAL_LOCAL_VAR
+set session binlog_space_limit=1;
+
+FLUSH LOGS;
+FLUSH LOGS;
+FLUSH LOGS;
+FLUSH LOGS;
+SHOW BINARY LOGS;
+
+FLUSH LOGS;
+# binary.000001 should be deleted now
+SHOW BINARY LOGS;
+
+CREATE TABLE `t1` (
+  `v1` int(11) DEFAULT NULL,
+  `v2` int(11) DEFAULT NULL,
+  KEY `v1` (`v1`)
+) ENGINE=MyISAM DEFAULT CHARSET=latin1;
```

Comment:
This would require include/have_myisam.inc, but any particular reason for a MyISAM table? If possible, just do not specify the engine